composer.json

```diff
@@ -21,7 +21,7 @@
         }
     },
     "require": {
-        "php": "7.3.*",
+        "php": "7.3.*|8.*",
```

Comment:
funderar på om vi ska skippa 8 for now, det är lite mer saker som behöver göras än bara php version